emcc.py

```diff
@@ -2313,6 +2313,13 @@ def parse_args(newargs):
         exit_with_error('Invalid value "' + newargs[i + 1] + '" to --output_eol!')
       newargs[i] = ''
       newargs[i + 1] = ''
+    elif newargs[i] == '--generate-config':
+      path = os.path.expanduser(newargs[i + 1])
```

Comment:
please save `newargs[i + 1]` to a local variable and use that

(we should do that in other places in this file too, sorry about that...)